libpkg/pkg_elf.c

```diff
@@ -846,6 +847,16 @@ pkg_get_myarch_elfparse(char *dest, size_t sz, int *osversion)
 	for (fd = -1, i = 0; i < nitems(abi_files); i++) {
 		if (abi_files[i] == NULL)
 			continue;
+		/*
+		 * Try prepending rootdir and using that if it exists.  If
+		 * ABI_FILE is specified, assume that the consumer didn't want
+		 * it mangled by rootdir.
+		 */
+		if (i > 0 && snprintf(rooted_abi_file, PATH_MAX, "%s/%s",
+		    ctx.pkg_rootdir, abi_files[i]) < PATH_MAX) {
```

Comment:
I only see pkg_rootdir set to / for scripts so that will ended up to (null)/ABI_FILES, maybe check that ctx.pkg_rootdir isn't NULL ?

Reply:
Whoops; fixed.